src/views/Simulation/components/Scene/MapView.js

```diff
@@ -0,0 +1,20 @@
+import React, { useEffect, useRef } from 'react';
+
+const MapView = () => {
+  const mapCanvasRef = useRef(null);
+  const mapAppRef = useRef(null);
+
+  useEffect(() => {
+    if (!mapCanvasRef.current) return;
+
+    return () => {
+      if (mapAppRef.current) {
+        mapAppRef.current.destroy(true, { children: true });
+      }
+    };
+  }, []);
```

Comment:
eslint warning to fix here. We can add `// eslint-disable-next-line react-hooks/exhaustive-deps` before the line using the ref to silent the warning 🙂